src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/original-pricing-plan-queries.mdx

```diff
@@ -130,7 +130,8 @@ If your organization is affected by this end-of-life and you'll be staying on ou
 Some details about querying usage data: 
 
 * Usage data is stored on the [`NrDailyUsage` event type](/attribute-dictionary/?event=NrDailyUsage) and the [`NrUsage` event type](/attribute-dictionary/?event=NrUsage)
-* Querying with the [query builder](/docs/query-your-data/explore-query-data/query-builder/introduction-query-builder) in the UI will query in a specific account, and any child accounts. You can also [use NerdGraph for cross-account querying](/docs/apis/nerdgraph/examples/nerdgraph-nrql-tutorial/#cross-account-query). 
+* If you're querying the `consumingAccount`, ensure you input the [account ID](/docs/accounts/accounts-billing/account-structure/account-id). 
+* Querying with the [query builder](/docs/query-your-data/explore-query-data/query-builder/introduction-query-builder) in the UI will query a specific account, and any child accounts. You can also [use NerdGraph for cross-account querying](/docs/apis/nerdgraph/examples/nerdgraph-nrql-tutorial/#cross-account-query). 
```

Comment:
@zuluecho9 to reduce repetition, you might consider this minor revision:

Using the [query builder](https://docswebsitedevelop-smallfixesjune26.gatsbyjs.io/docs/query-your-data/explore-query-data/query-builder/introduction-query-builder/) in the UI will query a specific account, and any child accounts.